pootle/templates/includes/formtable.html

```diff
@@ -12,24 +12,28 @@
 	  <div class="aligned-center pagination">
 	    <div class="pagination-left js-pagination-previous">
 	      {% if formtable.page.has_previous %}
-	      <a href="?page={{ formtable.page.previous_page_number }}">&lt;&lt;&lt;</a>
+	      <a href="?page={{ formtable.page.previous_page_number }}">{% trans "&lt;&lt;&lt;" %}</a>
 	      {% else %}
-	      <span class="disabled">&lt;&lt;&lt;</span>
+	      <span class="disabled">{% trans "&lt;&lt;&lt;" %}</span>
 	      {% endif %}
 	    </div>
 	    <div class="pagination-center">
 	      <span class="pagination-info js-pagination-page-no">
-		Page {{ formtable.page_no }} of {{ formtable.page.paginator.num_pages }}.
+		  {% blocktrans with page_number=formtable.page_no page_count=formtable.page.paginator.num_pages trimmed %}
+		  Page {{ page_number }} of {{ page_count }}.
+		  {% endblocktrans %}
 	      </span>
 	      <span class="pagination-info js-pagination-items-per-page">
-		{% blocktrans %}Showing{% endblocktrans %} {{ formtable.results_per_page }} {{ formtable.form.results_per_page.label }}
+		  {% blocktrans with count=formtable.results_per_page trimmed %}
+		  Showing {{ count }} results per page
+		  {% endblocktrans %}
 	      </span>
 	    </div>
 	    <div class="pagination-right js-pagination-next">
 	      {% if formtable.page.has_next %}
-	      <a href="?page={{ formtable.page.next_page_number }}">&gt;&gt;&gt;</a>
+	      <a href="?page={{ formtable.page.next_page_number }}">{% trans "&gt;&gt;&gt;" %}</a>
```

Comment:
I was thinking RTL. Or some language with its quirks regarding how to represent this action. But I will not oppose at all in dropping this. I even put it on a separate commit anticipating this possibility.